tests/test_state.py

```diff
@@ -165,7 +165,16 @@ def _state_notify_update(update, is_add_req):
                omap_state.OMAP_VERSION_KEY)
     assert (ioctx.notify(omap_state.omap_name))  # Send notify signal
 
-    time.sleep(0.5)
+    # any wait interval smaller than update_interval_sec = 10 should be good
+    # to test notify capability
     elapsed = time.time() - start
+    wait_interval = update_interval_sec - elapsed - 0.5
+    assert(wait_interval > 0)
+    assert(wait_interval < update_interval_sec)
+    time.sleep(wait_interval)
```

Comment:
I would suggest adding an info print  for wait_interval and update_interval_sec to track that on run (not a must for this PR)